pytest_fixtures/certificate.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+
+import pytest
+from fauxfactory import gen_string
+
+from robottelo import ssh
+from robottelo.config import settings
+from robottelo.helpers import get_data_file
+from robottelo.ssh import get_connection
+from robottelo.ssh import upload_file
+
+
+@pytest.fixture(scope="module")
+def cert_data():
+    """Get host name, scripts, and create working directory."""
+
+    def _cert_data(hostname=None):
+        sat6_hostname = hostname or settings.server.hostname
+        capsule_hostname = 'capsule.example.com'
+        key_file_name = '{0}/{0}.key'.format(sat6_hostname)
```

Comment:
May I suggest:  `'/'.join([sat6_hostname]*2)`

Reply:
You can use that for `base_name`, and then pass `f'{base_name}.key` and `f'{base_name}.crt'` below